### PR TITLE
docs(proposal): new artifacts distribution proposal

### PR DESCRIPTION
**Pull request by leogr**
**What type of PR is this?**

/kind documentation

**Any specific area of the project related to this PR?**

/area proposals

**What this PR does / why we need it**:

This proposal aims to formalize the revamped way adopted by Falcosecurity to distribute artifacts. In fact, it reflects some topics widely discussed in the last few months, plus suggestions for some obvious action items to move forward.

Furthermore, most of the points reflect some already ongoing efforts, such as the revamp of falcoctl.

Any feedback is appreciated

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

/milestone 0.34.0

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by leogr** [comment]
Dear @falcosecurity/core-maintainers 

As discussed during the last community call, here's my proposal :)
So, PTAL :pray: 

:hugs:

File: proposals/20221129-artifacts-distribution.md
Comment (FedeDP) [inline proposals/20221129-artifacts-distribution.md:72]:
```suggestion
Mirrors are allowed and encouraged if they facilitate artifacts consumption by our users. This proposal reccomends to enable mirrors on the major public OCI registry, such as [Amazon ECR](https://gallery.ecr.aws/) (which is already implentend in our infra at the time of writing).
```

**Review by Andreagit97 — APPROVED** [review]
/approve

**Review by FedeDP — APPROVED** [review]
/approve

**Comment by leogr** [comment]
As discussed in the current community call, it seems to me that nobody disagrees with this proposal, and we would like to deliver some action items within Falco 0.34, so

/hold cancel

:partying_face:

**Pull request merged**
